VideoLocker/src/main/java/org/edx/mobile/model/course/BlockType.java

```diff
@@ -4,6 +4,12 @@
  * Created by hanning on 5/21/15.
  */
 public enum  BlockType {
-    COURSE, CHAPTER,  SECTION,  SEQUENTIAL,
-    VERTICAL,  VIDEO,  HTML, PROBLEM, DISCUSSION, OTHERS
+    COURSE(false), CHAPTER(false),  SECTION(false),  SEQUENTIAL(false),
```

Comment:
This adds a bunch of data onto the enum values, which makes more sense to encapsulate as a separate method derived from the data. This is much more extensible for any future properties.

For example,

```
boolean isUnit() {
  switch(this) {
     case COURSE:
     case CHAPTER:
     ... other cases ...
         return false
    case HTML:
    case VIDEO:
     ... other cases ...
         return true
  }
}
```

Also, we probably shouldn't call this a "unit" since unit is also, unfortunately, used in some places in the (server only?) code to refer to a specific container depth.

Maybe more like ``isLeaf``?